CHANGELOG.rst

```diff
@@ -15,6 +15,10 @@ Unreleased
 ~~~~~~~~~~
 * Nothing
 
+[0.0.3] - 2018-02-13
```

Comment:
You have to bump the version number in completion/__init__.py.  Then I guess we should change this version number to match the new one.

Reply:
Done